docs/source/changes.rst

```diff
@@ -4,7 +4,7 @@ Changelog
 =========
 
 V1.15.0-dev (not released yet)
-------------------
+------------------------------
```

Comment:
das muss man uebrigens wieder zurueckaendern, sobald da in Klammern YYYY-MM-DD steht.

Reply:
nein, die Striche dürfen länger sein, aber nicht kürzer als die Überschrift

Reply:
ah, dann ist gut!